lib/msf/base/sessions/mettle_config.rb

```diff
@@ -9,6 +9,11 @@ module Msf
   module Sessions
     module MettleConfig
 
+      class << self
```

Comment:
Can probably be removed as superfluous: https://github.com/rapid7/metasploit-framework/pull/11903#discussion_r305939293